### PR TITLE
Featuredkits

### PR DESCRIPTION
**Pull request by jaimelr**
I added the basic structure for the featured kits section and also the content like text and images this section will have.

### CODE REVIEW

File: theme6/source/stylesheets/site.css.scss [hunk 1]
```diff
@@ -1,24 +1,160 @@
...
+        .contenedornews input{
+          height: 40px;
+          width: 305px;
+          border: 2px solid #D2D2D2;
+          border-radius: 2px;
+          background-color: #FBFBFB;
+        }
+        footer .author p{
+          margin-top: 35px;
+          font-size: 12px
```
Comment (magmahound) [inline theme6/source/stylesheets/site.css.scss]: Declaration should be terminated by a semicolon

File: theme6/source/stylesheets/site.css.scss [hunk 2]
```diff
@@ -1,24 +1,160 @@
...
+        .contenedornews{
+          margin-top: 5px;
+          display:block;
+          align-content: center;
+          width: 100%;
+        }
+        .contenedornews p{
+          font-size: 15px;
+        }
+        .contenedornews input{
```
Comment (magmahound) [inline theme6/source/stylesheets/site.css.scss]: Opening curly brace `{` should be preceded by one space

File: theme6/source/stylesheets/site.css.scss [hunk 3]
```diff
@@ -1,24 +1,160 @@
...
+          align-content:center;
+        }
+
+        .contenedornews{
+          margin-top: 5px;
+          display:block;
+          align-content: center;
+          width: 100%;
+        }
+        .contenedornews p{
```
Comment (magmahound) [inline theme6/source/stylesheets/site.css.scss]: Opening curly brace `{` should be preceded by one space

File: theme6/source/stylesheets/site.css.scss [hunk 4]
```diff
@@ -1,24 +1,160 @@
...
+        .box-social{align-content:center;}
+        .box-social p{font-size: 14px}
+        .boxsocialmedia{
+          width: 95%;
+          display:inline-flex;
+          justify-content:space-between;
+          align-content:center;
+        }
+
+        .contenedornews{
```
Comment (magmahound) [inline theme6/source/stylesheets/site.css.scss]: Opening curly brace `{` should be preceded by one space

File: theme6/source/stylesheets/site.css.scss [hunk 5]
```diff
@@ -1,24 +1,160 @@
...
+      #twt{
+        margin-left:-20px;
+        margin-right: -20px;}
+
+        .contenedorsocial{
+          display:flex;
+        margin-left: 50px;}
+        .box-social{align-content:center;}
+        .box-social p{font-size: 14px}
+        .boxsocialmedia{
```
Comment (magmahound) [inline theme6/source/stylesheets/site.css.scss]: Opening curly brace `{` should be preceded by one space

File: theme6/source/stylesheets/site.css.scss [hunk 6]
```diff
@@ -1,24 +1,160 @@
...
+      .logo{display:none;}
+      #twt{
+        margin-left:-20px;
+        margin-right: -20px;}
+
+        .contenedorsocial{
+          display:flex;
+        margin-left: 50px;}
+        .box-social{align-content:center;}
+        .box-social p{font-size: 14px}
```
Comment (magmahound) [inline theme6/source/stylesheets/site.css.scss]: Opening curly brace `{` should be preceded by one space<br>Declaration should be terminated by a semicolon

File: theme6/source/stylesheets/site.css.scss [hunk 7]
```diff
@@ -1,24 +1,160 @@
...
+    }
+/*=============Stylesheet MOBILE==========*/
+    @media screen and(max-width:320px) {
+      .information{display: none;}
+      .logo{display:none;}
+      #twt{
+        margin-left:-20px;
+        margin-right: -20px;}
+
+        .contenedorsocial{
```
Comment (magmahound) [inline theme6/source/stylesheets/site.css.scss]: Opening curly brace `{` should be preceded by one space

File: theme6/source/stylesheets/site.css.scss [hunk 8]
```diff
@@ -1,24 +1,160 @@
...
+
+    footer .author p {
+      color: #72808C;
+      font-family:'Montserrat', sans-serif;
+      font-size: 16px;
+      line-height: 18px;
+    }
+/*=============Stylesheet MOBILE==========*/
+    @media screen and(max-width:320px) {
+      .information{display: none;}
```
Comment (magmahound) [inline theme6/source/stylesheets/site.css.scss]: Opening curly brace `{` should be preceded by one space

File: theme6/source/stylesheets/site.css.scss [hunk 9]
```diff
@@ -1,24 +1,160 @@
...
+        .boxsocialmedia{
+          width: 95%;
+          display:inline-flex;
+          justify-content:space-between;
+          align-content:center;
+        }
+
+        .contenedornews{
+          margin-top: 5px;
+          display:block;
```
Comment (magmahound) [inline theme6/source/stylesheets/site.css.scss]: Colon after property should be followed by one space

File: theme6/source/stylesheets/site.css.scss [hunk 10]
```diff
@@ -1,24 +1,160 @@
...
+        .contenedorsocial{
+          display:flex;
+        margin-left: 50px;}
+        .box-social{align-content:center;}
+        .box-social p{font-size: 14px}
+        .boxsocialmedia{
+          width: 95%;
+          display:inline-flex;
+          justify-content:space-between;
+          align-content:center;
```
Comment (magmahound) [inline theme6/source/stylesheets/site.css.scss]: Colon after property should be followed by one space

File: theme6/source/partials/_footer.erb [hunk 11]
```diff
@@ -0,0 +1,40 @@
+<div class="contenedor">
+      <footer>         
+          <div class="contenedorsocial">
```
Comment (albertm805) [inline theme6/source/partials/_footer.erb]: When adding a name to a class, try and do it in English. I see that you have `boxsocialmedia` below, which is in English. So, in order to keep the consistency name all your classes in English. Also, it is good practice for your English skills.

File: theme6/source/partials/_footer.erb [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+            <div class="box-social">
+              <p class="socialmedia" i>FOLLOW US</p>
+              <div class="boxsocialmedia">
```
Comment (albertm805) [inline theme6/source/partials/_footer.erb]: How come you didn't use dashes here like you did in `box-social`. This class can be renamed like this `box-social-media` or `box-socialmedia`. Whichever is fine.

File: theme6/source/partials/_offers.erb [hunk 13]
```diff
@@ -0,0 +1,34 @@
+<section class="featured-kits">
```
Comment (albertm805) [inline theme6/source/partials/_offers.erb]: Nice partial 👍 Very simple and clean. Keep doing this. As partials should not be too extensive.

File: theme6/source/stylesheets/main.css [hunk 14]
```diff
@@ -0,0 +1,280 @@
...
+.mid-banner-saylor {
+  background-image: url("../images/mid_banner1.png");
+  background-size: 130%;
+  background-repeat: no-repeat;
+  background-position: top;
+}
+
+.mid-banner-archeologist {
+  background-image: url("../images/mid_banner2.png");
+  background-size: 130%;
```
Comment (albertm805) [inline theme6/source/stylesheets/main.css]: Remember percentages, should almost never be used. An alternative to this is to use padding or positioning the image in your container.

File: theme6/source/stylesheets/main.css [hunk 15]
```diff
@@ -0,0 +1,280 @@
...
+    margin-bottom: 30px;
+  }
+
+  .big-banner-content h1 {
+    font-size: 32px;
+  }
+
+  .big-banner-content button {
+    padding-left: 50px;
+    padding-right: 50px;
```
Comment (albertm805) [inline theme6/source/stylesheets/main.css]: Just a quick tip, no need to make changes, but you can also add this to one line. `padding: 0 50px;`

File: theme6/source/stylesheets/main.css [hunk 16]
```diff
@@ -0,0 +1,280 @@
...
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .mid-banner {
+    flex: 1;
+    background-size: 60%;
```
Comment (albertm805) [inline theme6/source/stylesheets/main.css]: 🤔 Do you really need a percentage here?

File: theme6/source/stylesheets/site.css.scss [hunk 17]
```diff
@@ -1,24 +1,159 @@
...
+    footer {
+      bottom: 0;
+      display: inline;
+      width: 100%;
+      
+    }
+
+    .information{justify-content: center;}
+
+    li{display: inline;}
```
Comment (albertm805) [inline theme6/source/stylesheets/site.css.scss]: If you are going to keep it in one line add spacing to it like so: `li { display: inline; }`

File: theme6/source/stylesheets/site.css.scss [hunk 18]
```diff
@@ -1,24 +1,159 @@
...
-}
-
-.middleman-logo {
-  margin-bottom: 1rem;
-  width: 10rem;
-}
+
+
+/*==========Stylesheet DESKTOP========*/
+    footer {
```
Comment (albertm805) [inline theme6/source/stylesheets/site.css.scss]: You have this indented..... I don't think it is necessary if you are just starting to write the styles

File: theme6/source/stylesheets/site.css.scss [hunk 19]
```diff
@@ -1,24 +1,159 @@
...
+      bottom: 0;
+      display: inline;
+      width: 100%;
+      
+    }
+
+    .information{justify-content: center;}
```
Comment (albertm805) [inline theme6/source/stylesheets/site.css.scss]: Also add spacing

File: theme6/source/stylesheets/site.css.scss [hunk 20]
```diff
@@ -1,24 +1,159 @@
...
+
+    footer .box-social button:hover {
+      cursor: pointer;
+    }
+
+    footer .contenedorsocial {
+      display: inline-flex;
+      justify-content: center;
+      align-items: center;
+      width: 66%;
```
Comment (albertm805) [inline theme6/source/stylesheets/site.css.scss]: Another `width` that looks fishy. `66%` 🤔 Why so much?

File: theme6/source/stylesheets/site.css.scss [hunk 21]
```diff
@@ -1,24 +1,159 @@
...
+    footer .contenedorsocial .logo {
+      margin-left: 96px;
+      margin-right: 96px;
+    }
+
+    footer .box-social p{
+      text-align: center;
+    }
+    footer .contenedornews{
+      width: 33%;
```
Comment (albertm805) [inline theme6/source/stylesheets/site.css.scss]: And another one here, can you do this another way than adding percentages please.

File: theme6/source/stylesheets/site.css.scss [hunk 22]
```diff
@@ -1,24 +1,159 @@
...
+      justify-content: center;
+    }
+
+    footer .information ul li a{
+      height: 22px;
+      width: 192px;
+      color: #72808C;
+      font-family:'Montserrat', sans-serif;
+      font-size: 18px;
+      line-height: 21px;
```
Comment (albertm805) [inline theme6/source/stylesheets/site.css.scss]: Some of your properties you have in alphabetical order and others you don't be a bit more consistent with this and keep them in order please.

File: theme6/source/stylesheets/site.css.scss (hunk 8 above)
Comment (albertm805) [reply theme6/source/stylesheets/site.css.scss]: As it says. Add spacing when opening and closing curly braces.

File: theme6/source/stylesheets/site.css.scss (hunk 7 above)
Comment (albertm805) [reply theme6/source/stylesheets/site.css.scss]: Space after `social` and the `{`

File: theme6/source/stylesheets/site.css.scss [hunk 23]
```diff
@@ -1,24 +1,159 @@
...
+      .logo{display:none;}
+      #twt{
+        margin-left:-20px;
+        margin-right: -20px;}
+
+        .contenedorsocial{
+          display:flex;
+        margin-left: 50px;}
+        .box-social{align-content:center;}
+        .box-social p{font-size: 14px}
```
Comment (albertm805) [inline theme6/source/stylesheets/site.css.scss]: You forgot to add a semicolon after the `14px;` like so.

File: theme6/source/stylesheets/site.css.scss [hunk 24]
```diff
@@ -1,24 +1,159 @@
...
+        .boxsocialmedia{
+          width: 95%;
```
Comment (albertm805) [inline theme6/source/stylesheets/site.css.scss]: Another percentage, is this really needed?

File: theme6/source/stylesheets/main.css [hunk 25]
```diff
@@ -0,0 +1,280 @@
+.featured-kits {
+  font-family: 'Oswald', sans-serif;
+}
+
+.big-banner {
+  background-image: url("../images/banner1_mobile.png");
+  background-size: 120%;
```
Comment (albertm805) [inline theme6/source/stylesheets/main.css]: Percentage is it really necessary?

File: theme6/source/stylesheets/main.css [hunk 26]
```diff
@@ -0,0 +1,280 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  height: 300px;
+  text-align: center;
+}
+
+.mid-banner-saylor {
+  background-image: url("../images/mid_banner1.png");
+  background-size: 130%;
```
Comment (albertm805) [inline theme6/source/stylesheets/main.css]: Another percentage.

**Review by albertm805 — CHANGES_REQUESTED** [review]
Made a couple of comments on your percentages that you are using. Also, you have a couple of comments that you need to address from the magmahound. If you need any help with any of this don;t hesitate in asking.

File: theme6/source/stylesheets/site.css.scss [hunk 27]
```diff
@@ -1,24 +1,162 @@
...
+      #twt{
+        margin-left:-20px;
+        margin-right: -20px;}
+
+        .contenedorsocial{
+
+          display:flex;
+        margin-left: 50px;}
+        .box-social{align-content:center;}
+        .box-social p{font-size: 14px}
```
Comment (magmahound) [inline theme6/source/stylesheets/site.css.scss]: Opening curly brace `{` should be preceded by one space<br>Declaration should be terminated by a semicolon

File: theme6/source/stylesheets/site.css.scss [hunk 28]
```diff
@@ -1,24 +1,162 @@
...
+    footer .author p {
+      color: #72808C;
+      font-family:'Montserrat', sans-serif;
+      font-size: 16px;
+      line-height: 18px;
+    }
+/*=============Stylesheet MOBILE==========*/
+    @media screen and(max-width:320px) {
+
+      .information{display: none;}
```
Comment (magmahound) [inline theme6/source/stylesheets/site.css.scss]: Opening curly brace `{` should be preceded by one space

File: theme6/source/partials/_footer.erb (hunk 11 above)
Comment (8geonirt) [reply theme6/source/partials/_footer.erb]: :fuckyeah: 🤓

File: theme6/source/partials/_offers.erb [hunk 29]
```diff
@@ -0,0 +1,34 @@
...
+      <div class="big-text">
+        <h1>ALL KITS</h1>
+        <h1>50% OFF</h1>
+      </div>
+    </div>
+  </div>
+  <div class="flex-grid">
+    <div class="mid-banner mid-banner-saylor">
+      <div class="mid-banner-content">
+        <h3>SAILORS & PIRATES KIT</h3>
```
Comment (8geonirt) [inline theme6/source/partials/_offers.erb]: Let's replace special characters using HTML entities like this:
`&amp;`

File: theme6/source/stylesheets/site.css.scss [hunk 30]
```diff
@@ -1,24 +1,162 @@
...
+        line-height: 21px;
+    }
+
+    .sm {
+        height: 35px;
+        width: 93px;
+        border: 2px solid #72808C;
+        border-radius: 3px;
+    }
+    #fb{
```
Comment (Ilyeo) [inline theme6/source/stylesheets/site.css.scss]: don't use ids to give styles, can you change ids by classes?

File: theme6/source/stylesheets/site.css.scss [hunk 31]
```diff
@@ -1,24 +1,162 @@
...
+      background: url("../images/fb.svg");
+      background-position: center center;
+      background-repeat: no-repeat;
+      margin-right: 23px;
+    }
+    #twt{
```
Comment (Ilyeo) [inline theme6/source/stylesheets/site.css.scss]: here too please

File: theme6/source/stylesheets/site.css.scss [hunk 32]
```diff
@@ -1,24 +1,162 @@
...
+        background: url("../images/twt.svg");
+        background-position: center center;
+        background-repeat: no-repeat;}
+    #g{
```
Comment (Ilyeo) [inline theme6/source/stylesheets/site.css.scss]: and here

**Review by Ilyeo — CHANGES_REQUESTED** [review]
beside my comments LGTM

**Pull request closed without merging**